lib/rules/no-passed-in-event-handlers.js

```diff
@@ -40,7 +40,53 @@ const EVENT_HANDLER_METHODS = new Set([
   'drop',
 ]);
 
+const DEFAULT_CONFIG = {
+  ignore: [],
+};
```

Comment:
This seems wrong now that we expect an object. I think the only reason it isn't throwing an error in the tests is that `let arr = []; arr.click` is perfectly "valid" (it'll just always be undefined).

Reply:
It doesn't seem wrong, it _is_ wrong. Updated.